pages/common/secrethub.md

```diff
@@ -0,0 +1,24 @@
+# secrethub
+
+> A tool to keep secrets out of config files.
+> More information: <https://secrethub.io>.
+
+- Print a secret to stdout:
+
+`secrethub read {{path/to/secret}}`
+
+- Generate a random value and store it as a new secret (version):
+
+`secrethub generate {{path/to/secret}}`
+
+- Store a value from the clipboard as a new secret (version):
+
+`secrethub write --clip {{path/to/secret}}`
+
+- Store a value supplied on stdin as a new secret (version):
+
+`echo "secret value" | secrethub write {{path/to/secret}}`
```

Comment:
Perhaps we could put `secret value` in the token syntax here please?
```suggestion
`echo "{{secret value}}" | secrethub write {{path/to/secret}}`
```

Reply:
```suggestion
`echo "{{secret_value}}" | secrethub write {{path/to/secret}}`
```

Should be with an underscore.

Reply:
Updated my suggestion :+1:

Reply:
Thanks! I've updated the example :+1: